### PR TITLE
Update panels docs getting started

### PR DESCRIPTION
**Pull request by Z3d0X**
- ~Replace the reference to `€` with `$`, as all the following documentation use USD/$~
- Fix minor inconsistency in widget docs
- Replace references to `$` with `€`

### CODE REVIEW

**Comment by zepfietje** [comment]
We should use € everywhere instead of $ because of a bug in Markdown rendering.

**Comment by danharrin** [comment]
So can you make it consistent with `€` instead?

File: packages/panels/docs/02-getting-started.md [hunk 1]
```diff
@@ -598,7 +598,7 @@ class Treatment extends Model
 
 ### Setting up the treatments table
 
-When the relation manager was generated previously, the `description` text column was automatically added. Let's also add a `sortable()` column for the `price` with a currency prefix. Use the Filament `money()` method to format the `price` column as money — in this case for `usd` (`$`):
+When the relation manager was generated previously, the `description` text column was automatically added. Let's also add a `sortable()` column for the `price` with a currency prefix. Use the Filament `money()` method to format the `price` column as money — in this case for `euro` (`€`):
```
Comment (zepfietje) [inline packages/panels/docs/02-getting-started.md]:
```suggestion
When the relation manager was generated previously, the `description` text column was automatically added. Let's also add a `sortable()` column for the `price` with a currency prefix. Use the Filament `money()` method to format the `price` column as money — in this case for `EUR` (`€`):
```

File: packages/panels/docs/02-getting-started.md [hunk 2]
```diff
@@ -610,7 +610,7 @@ public function table(Table $table): Table
         ->columns([
             Tables\Columns\TextColumn::make('description'),
             Tables\Columns\TextColumn::make('price')
-                ->money('usd')
+                ->money('euro')
```
Comment (zepfietje) [inline packages/panels/docs/02-getting-started.md]:
```suggestion
                ->money('EUR')
```

File: packages/panels/docs/02-getting-started.md [hunk 3]
```diff
@@ -628,7 +628,7 @@ public function table(Table $table): Table
...
```
Comment (zepfietje) [inline packages/panels/docs/02-getting-started.md]:
```suggestion
                ->money('EUR')
```

File: packages/panels/docs/02-getting-started.md [hunk 4]
```diff
@@ -686,7 +688,7 @@ Let's add a chart to the dashboard to visualize the number of treatments adminis
 php artisan make:filament-widget TreatmentsChart --chart
 ```
 
-When prompted, select "admin" for the location and "line chart" as the chart type.
+When prompted, do not specify a resource, select "admin" for the location and "line chart" as the chart type.
```
Comment (zepfietje) [inline packages/panels/docs/02-getting-started.md]:
```suggestion
When prompted, do not specify a resource, select "admin" for the location, and choose "line chart" as the chart type.
```

File: packages/panels/docs/02-getting-started.md [hunk 5]
```diff
@@ -646,12 +646,14 @@ Let's add a stats widget to our default dashboard page that includes a stat for
...
 ### Creating a stats widget
 
-Create a [stats widget](../widgets/stats-overview) to render patient types using the following artisan command (do not specify a resource and select "admin" for the location when prompted):
+Create a [stats widget](../widgets/stats-overview) to render patient types using the following artisan command:
 
 ```bash
 php artisan make:filament-widget PatientTypeOverview --stats-overview
 ```
 
+When prompted, do not specify a resource, select "admin" for the location.
```
Comment (zepfietje) [inline packages/panels/docs/02-getting-started.md]:
```suggestion
When prompted, do not specify a resource, and select "admin" for the location.
```

**Comment by zepfietje** [comment]
Thanks!

**Comment by zepfietje** [comment]
Thanks!

**Pull request merged**